src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
+import Nav from './components/Nav';
+import Book from './components/Book';
+import Catagories from './components/catagory';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Nav />
+      <Routes>
+        <Route exact path="/" element={<Book />} />
+        <Route exact path="catagory" element={<Catagories />} />
```

Comment:
- [ ] According to the project requirements, please kindly ensure that this route is `/categories`.